pkg/webhook/package_webhook.go

```diff
@@ -127,11 +122,17 @@ func validatePackage(p *v1alpha1.Package, jsonSchema []byte) (*gojsonschema.Resu
 		return nil, fmt.Errorf("error compiling schema %v", err)
 	}
 
-	packageConfig, err := yaml.YAMLToJSON([]byte(p.Spec.Config))
+	packageConfigBytes, err := yaml.YAMLToJSON([]byte(p.Spec.Config))
```

Comment:
Seems a bit odd to parse it, change it to a string and then ignore all that if the thing was blank, but I can see how this would work